### ISSUE TITLE
"attr-new-line": 0 issue with latest version 0.5.1

### ISSUE THREAD

**Issue by nghiado** [linthtml/linthtml#273]
I upgraded  "@linthtml/linthtml": "^0.3.2", to "5.0.1" and got many errors on existing code base.
Here is the sample html template
```html
<div
  class="wizard-navigator"
>
  <div
    class="page-titles"
  >
```
but still get error  
```Only 0 attributes per line are permitted  attr-new-line```

Confirmed that version  "@linthtml/linthtml": "^0.4.1" working fine with "attr-new-line": 0 setting

**Comment by KamiKillertO** [comment]
@nghiado thanks for reporting, I'll check this.

> I upgraded "@linthtml/linthtml": "^0.3.2", to "5.0.1" and got many errors on existing code base.

Many errors that are not correct?

**Comment by nghiado** [comment]
@KamiKillertO , I double checked with 0.4.1 and htmllint pass with the same config/html files.

**Comment by KamiKillertO** [comment]
The HTML parser used by linthtml has been updated in the release 0.5.0. Thanks to that some errors that were not detected before are detected now. So, it's not abnormal if you have new errors after updating but it's also possible for some bugs to have been introduced 😓.
So please report issues if you thinks that some of your errors are incorrect 🙏

**Comment by nghiado** [comment]
@KamiKillertO As you can see, this template should pass lint rule **"attr-new-line": 0** acording to the doc

_If set, no more than this number of attributes may be on the same row. A value of 0 applies only to the first row, restricting subsequent rows to one element._

```
<div
  class="wizard-navigator"
>
</div>
```

**Comment by KamiKillertO** [comment]
@nghiado I've just published version 0.5.2 with the fix for your issue. Please report any other errors you might encounter in the future 😄

**Comment by nghiado** [comment]
Thank you very much @KamiKillertO
